creme/creme_core/models/auth.py

```diff
@@ -326,6 +326,20 @@ def filter_entities(self,
             as_model=as_model,
         )
 
+    def refresh_from_db(self, *args, **kwargs):
+        self._allowed_apps = None
+        self._extended_allowed_apps = None
+
+        self._admin_4_apps = None
+        self._extended_admin_4_apps = None
+
+        self._creatable_ctypes_set = None
+        self._exportable_ctypes_set = None
+
+        self._setcredentials = None
+
+        super().refresh_from_db(*args, **kwargs)
+
```

Comment:
Unit test (I presume it's better to test that queries are done with assertNumQueries() than testing protected attributes).